src/java/picard/analysis/CollectOxoGMetrics.java

```diff
@@ -40,12 +40,34 @@
  * Class for trying to quantify the CpCG->CpCA error rate.
  */
 @CommandLineProgramProperties(
-        usage = CollectOxoGMetrics.USAGE,
-        usageShort = CollectOxoGMetrics.USAGE,
+        usage = "Collects metrics quantifying the error rate resulting from oxidative artifacts <br /><br />" +
+                "Oxidation of guanine to 8-oxoguanine can occur during library preparation of genomic DNA via a " +
+                "combination of heat, shearing, and metal contaminates in a sample (doi: 10.1093/nar/gks1443).  " +
+                "In DNA, 8-oxoguanine can pair with " +
+                "either cytosine or adenine, ultimately leading to G→T transversion mutations during PCR amplification.  " +
+                "The variants (C→A)/(G→T) tend to occur in specific sequence contexts e.g. CCG→CAG" +
```

Comment:
The "→" symbol may not be widely supported, should either use non-special characters or look up the html code equivalents.